pkg/nfd-gc/nfd-gc.go

```diff
@@ -37,6 +39,10 @@ import (
 	"sigs.k8s.io/node-feature-discovery/pkg/version"
 )
 
+const (
+	NRTOwnerPodLabel = "nfd.node.kubernetes.io/owner-pod"
```

Comment:
nit: I think the "owner" of the label name is the nfd-topology-updater so this would be better to move in `pkg/nfd-topology-updater/nfd-topology-updater.go`. Makes sense?